server/core/config/raw/apply_settings.go

```diff
@@ -0,0 +1,40 @@
+package raw
+
+import (
+	validation "github.com/go-ozzo/ozzo-validation"
+	"github.com/runatlantis/atlantis/server/core/config/valid"
+)
+
+type ApplySettings struct {
+	PRRequirements    []string `yaml:"pr_requirements" json:"pr_requirements"`
+	BranchRestriction string   `yaml:"branch_restriction" json:"branch_restriction"`
+	Team              string   `yaml:"team" json:"team"`
+}
+
+func (s ApplySettings) Validate() error {
+	return validation.ValidateStruct(&s,
+		validation.Field(&s.PRRequirements, validation.By(func(value interface{}) error {
+			v := value.([]string)
+
+			for _, item := range v {
+				return validation.In(valid.ApprovedApplyReq).Validate(item)
+			}
+			return nil
+		})),
+		validation.Field(&s.BranchRestriction, validation.In(string(valid.NoBranchRestriction), string(valid.DefaultBranchRestriction))),
+		validation.Field(&s.Team),
```

Comment:
Just confirming there intentionally is no validation rule for `s.Team` here right?

Reply:
yeah i don't think there's anything to validate.  I guess I can just remove it.